### PR TITLE
Update ion.rangeslider version, directive and 'How to use..' in README

### PR DESCRIPTION
**Pull request by agustin107**
#16 
This was updated about ion.rangeSlider specification of new version, that need to use was input base and no an element.

### CODE REVIEW

**Comment by geoffreybauduin** [comment]
Thank you for this awesome contribution.

I'm a bit concerned about one change though, more in comments

File: ionic-range-slider.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ angular.module("ion.rangeslider", []);
 angular.module("ion.rangeslider").directive("ionRangeSlider",
     function($timeout) {
         return {
-            restrict: "E",
+            restrict: "A",
```
Comment (geoffreybauduin) [inline ionic-range-slider.js:10]: That change is a big breaking change. Maybe we should bump a little more the version to `1.1` or something like that

Comment (agustin107) [reply ionic-range-slider.js:10]: Yep, you have reason. Then, What is the better version for you about this change? 1.1 is better for you?
I you agree, I gonna change that.

Comment (geoffreybauduin) [reply ionic-range-slider.js:10]: Yes, let's go for 1.1 !

Comment (agustin107) [reply ionic-range-slider.js:10]: Perfect!

**Pull request merged**